### ISSUE TITLE
Needed sounds for many mods

### ISSUE THREAD

**Issue by Ombridride** [Ombridride/minetest-minetestforfun-server#46]
Coming soon

**Comment by Ombridride** [comment]
@Cyberpangolin and @ObaniGemini proposed to do parts of this sounds, your help is welcome ! :)

**Comment by Ombridride** [comment]
Up, no one wants new monsters sounds for more realism ? :)
@Cyberpangolin @ObaniGemini @davedevils @gravgun

**Comment by Cyberpangolin** [comment]
I have a bit of time these days... Stay tuned, it might come next week. 
Maybe.

**Comment by Cyberpangolin** [comment]
War_cry : When ???

I think it should be at the moment the mob sees the player and starts focusing.
Maybe with a probability not at 100% to keep sometimes the surprise.

**Comment by Ombridride** [comment]
The **War_cry** is already implemented, i just haven't investigate the code to check when the **War_cry** is activated

**Comment by Cyberpangolin** [comment]
Ok, I will try to make one mob per day, until all are completed.
The first one comes tomorrow.

**Comment by Ombridride** [comment]
Good news, can't wait to hear that ! :)

**Comment by Cyberpangolin** [comment]
First, the male NPC.
Note, all files are stored in minetest-minetestforfun-server/other_things/mobsounds/<Mob_Name>
Please listen to them, tell me what to improve if needed, and if you like them don't hesitate to integrate them ingame. You always will get: the separated files and the original audacity file.

licence (if needed) WTFPL

**Comment by Cyberpangolin** [comment]
Today, minothaur.
For an obscure reason, an object in homedecor seems to have been modified. I did not touch it, so I suppose it's due to the chmod command.

Anyway, feel free to listen to them and include them ingame -- if you like -- , or to modifiy them.

**Comment by Ombridride** [comment]
Up, @Cyberpangolin and @ObaniGemini can you create some mobs sounds for us ? :)

**Comment by ObaniGemini** [comment]
I do not create, I remake :p

**Comment by Cyberpangolin** [comment]
Sorry, I don't have much time... Yes, I really WANT to make them, and I will.
On the next week I can promise to give 5 hours on it, I have planned them now.

**Comment by Cyberpangolin** [comment]
I added yesterday evening a try for the female npc. Note I can still record more voice for her, what we have is mainly laughings and damage taken.

**Comment by Cyberpangolin** [comment]
I have added now yeti sounds.

**Comment by Ombridride** [comment]
Great, thank you

**Comment by Cyberpangolin** [comment]
I don't think so, all are in mono, so the compression should not be that important...

**Comment by Ombridride** [comment]
@Cyberpangolin 
The sound *mobs_fnpc_random_1.ogg* is maybe bugged and its size is 10 times more than the other random sounds with no reason, it's strange...

**Comment by Cyberpangolin** [comment]
I corrected this in https://github.com/MinetestForFun/server-minetestforfun/commit/7c5606c747b6c8c3bf3ec866b5037733b0a255b0.
 sorry... Small rushes were at the end of the file. Now it should work better :+1:

**Comment by ObaniGemini** [comment]
Have fun, I had too :)
https://up1.ca/#swDcfdlfZsWTtWTtjnljjQ

**Comment by Ombridride** [comment]
Thank you for this sounds Obani, it's interresting because with the Obani and Cyberpangolin sounds, i think i can do really interresting Yeti sounds, I will put time on this soon :)

**Comment by Cyberpangolin** [comment]
For a small movie done with a child, we have done these sounds. They are so great I share them :)
WTFPL as always
[trex.zip](https://github.com/MinetestForFun/server-minetestforfun/files/320917/trex.zip)

**Comment by Darcidride** [comment]
Hi everyone, 

In a way to reduce the server lag and RAM needed by client, I was thinking of something...

#### The idea
Actually for the "**Damage** (When the mob suffers damage)" parameter, mobs_redo use a **default** sound if no specific sound are set, that's a good idea, with this we can have **Damage** sounds every time a player punch a mob, with or without specific sound for the mob.

### Great explanation, but what is my idea ? 
Simply, if we can find/create a sound for each **default** sound possibility, we will surely be able to erase some sound **and keep** the actual rich sound glossary of our server!

### Sound list that needs a "default" sound
Attack : NOK
Damage : OK
Death : NOK
Random : NOT NEEDED, too specific and give great atmosphere with the specific mob
War_cry : NOT NEEDED, too specific and give great atmosphere with the specific mob

**Comment by ObaniGemini** [comment]
:+1:
